tools-tested-by-miwg.json

```diff
@@ -116,15 +116,14 @@
       "vendor": "Camunda",
       "tool": "bpmn.io (Camunda Modeler)",
       "maintainer": "nele.uhlemann(at)camunda.com",
-      "version": "9.4.0",
       "website": "https://camunda.com/bpmn/tool/",
       "bpiId": "",
       "supportsBpmn2": true,
       "hasImport": true,
       "hasExport": true,
       "hasRoundtrip": true,
-      "lastResultsSubmitted": "2022",
-      "participatedInDemosByYear": "2022,2020,2019,2018,2017,2016,2015,2014,2013",
+      "lastResultsSubmitted": "2023",
+      "participatedInDemosByYear": "2023,2022,2020,2019,2018,2017,2016,2015,2014,2013",
```

Comment:
@barmac There was no demo this year. So we should not yet add an entry to `participatedInDemosByYear`